### PR TITLE
Use secrets module

### PR DESCRIPTION
**Pull request by willianpaixao**
As per Python [documentation]([url](https://docs.python.org/3/library/secrets.html)):

> In particular, secrets should be used in preference to the default pseudo-random number generator in the random module, which is designed for modeling and simulation, not security or cryptography.

> The secrets module provides access to the most secure source of randomness that your operating system provides.

Additionally:
- Setting English as the default language straight in the __init__ function
- Checking for the existence of the dictionary file before attempting to open it

### CODE REVIEW

File: src/mnemonic/mnemonic.py [hunk 1]
```diff
@@ -65,28 +66,26 @@ def b58encode(v: bytes) -> str:
...
-        return os.path.join(os.path.dirname(__file__), "wordlist")
+        f = os.path.join(os.path.dirname(__file__), f"wordlist/{language}.txt")
+        if os.path.exists(f) and os.path.isfile(f):
+            with open(f, "r", encoding="utf-8") as f:
+                self.wordlist = [w.strip() for w in f.readlines()]
+            if len(self.wordlist) != self.radix:
+                raise ConfigurationError(
+                    "Wordlist should contain %d words, but it contains %d words."
+                    % (self.radix, len(self.wordlist))
+                )
```
Comment (prusnak) [inline src/mnemonic/mnemonic.py:80]: Since you are using format strings in this PR, please convert this too.

**Review by prusnak — CHANGES_REQUESTED** [review]

File: src/mnemonic/mnemonic.py (hunk 1 above)
Comment (willianpaixao) [reply src/mnemonic/mnemonic.py]: Done.

**Comment by matejcik** [comment]
> Let's drop 3.5 and 3.6 (EOL in a month).

this must be done in CI and in `setup.py` (to set higher minimum version)

**Comment by willianpaixao** [comment]
@matejcik @prusnak 
- [X] Separate different tasks in different commits
- [X] Drop support to Python 3.5 and 3.6

**Comment by matejcik** [comment]
thanks!

**Pull request merged**